apps/api/v2/src/ee/event-types/event-types_2024_04_15/inputs/update-event-type.input.ts

```diff
@@ -177,6 +177,14 @@ export class BookingField_2024_04_15 {
   @ValidateNested({ each: true })
   @Type(() => Source)
   sources?: Source[];
+
```

Comment:
I see that inputs and outputs of event-types in folder `event-types_2024_04_15` are updated, but we also need to update inputs and outputs used by next release in the folder `event-types_2024_06_14`

The controller `apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types.controller.ts` takes inputs from `packages/platform/types/event-types/event-types_2024_06_14/inputs` and outputs from `apps/api/v2/src/ee/event-types/event-types_2024_06_14/outputs` that share underlying event-type output defined in `packages/platform/types/event-types/event-types_2024_06_14/outputs/event-type.output.ts`.

As you see, the inputs and outputs are in the `platform-types` package , which means that after changing them you have to re-build them by running `rm -rf dist && npx tsc --build --force` in the `packages/platform/types` folder.